app/assets/javascripts/controllers/schedule/schedule_form_controller.js

```diff
@@ -33,17 +32,18 @@ miqAngularApplication.controller('scheduleFormController', ['$http', '$scope', '
       $scope.newRecord                         = true;
       $scope.scheduleModel.action_typ          = 'vm';
       $scope.scheduleModel.filter_typ          = 'all';
-      $scope.scheduleModel.enabled             = '1';
-      $scope.scheduleModel.filterValuesEmpty   = true;
-      var today                                 = new Date();
-      var tomorrowsDate                         = parseInt(today.getDate()) + 1;
-      $scope.scheduleModel.miq_angular_date_1  = today.getMonth() + 1 + "/" + tomorrowsDate + "/" + today.getFullYear();
+      $scope.scheduleModel.enabled             = true;
+      $scope.filterValuesEmpty   = true;
+      var today                                = new Date();
+      var tomorrowsDate                        = parseInt(today.getDate()) + 1;
+      $scope.scheduleModel.start_date          = new Date(today.getMonth() + 1 + "/" + tomorrowsDate + "/" + today.getFullYear());
```

Comment:
This is broken - there is *no* guarantee `new Date` can parse that kind of date string (only RFC and ISO formats are required to be supported), and either way, it doesn't work on the last day of a month.

Since we're going to be handling dates more and more in JS, maybe we could consider using [moment.js](http://momentjs.com/) or something similar?